### ISSUE TITLE
Wrong Overfitting Tests

### ISSUE THREAD

**Issue by mravanelli** [speechbrain/speechbrain#111]
In all the current minimal examples, we check the final PER on the test set rather than on the training set. To be a significant test, it should be done on the training loss (test loss can be arbitrary high, but the training one should be very small because we can "memorize" the data within the neural network).
